gpt_pp/ai/__init__.py

```diff
@@ -89,15 +89,15 @@ def get_chat(self, files: str) -> ConversationChain:
             memory=memory,
```

Comment:
The updated spinner text "Reviewing pull request" is more appropriate for the task being performed. Nice improvement.